Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -551,7 +551,9 @@ def main():
     proxy = params.get('proxy', False)
     command = demisto.command()
     args = demisto.args()
-
+    if params.get('feedIncremental') and params.get('feedExpirationPolicy') == 'suddenDeath':
+        raise DemistoException('In order to use the feedIncremental parameter, the feedExpirationPolicy must be set to '
+                               'a value other than suddenDeath.')
```

Comment:
feedIncremental is hidden for the customer and will always be true so instead we can just do:
```suggestion
    if params.get('feedExpirationPolicy') == 'suddenDeath':
        raise DemistoException('The feed is incremental, so a sudden-death policy is not applicable.')
```